usefulUtilities/officeMat/README.md

```diff
@@ -0,0 +1,42 @@
+# Office Mat
+This project folder contains a number of entity scripts that attach to an Office Mat, Office Mat Squares, or the zones that surround the Office Mat Squares. These entity scripts do cool things.
+
+# Features
+- When you step onto a red, yellow, or grey Office Mat Squares, a script will update your status on the High Fidelity Status Indicator database to "busy". Alternatively, if configured, the script will update your status to some custom text.
+- When you step onto a green Office Mat Square, a script will update your status on the High Fidelity Status Indicator database to "available". Alternatively, if configured, the script will update your status to some custom text.
+
+# Office Mat Setup
+1. Create a `<matSquares Folder>/config/config.json` file that contains the data below.
+2. Rez a Box entity to act as your "Office Mat".
+3. Rez another Box entitity to act as your first "Office Mat Square". You will stand on these Squares to update your in-world status.
+4. Rez a Zone entity and position it so that you are inside that zone when you step inside the Square you just rezzed. Parent this Zone entity to your Office Mat Square from (2).
+5. (Optional) Rez a Text entity and position it so that the text is readily associated with the status of that square. Parent this Text entity to your Office Mat Square from (2).

```

Comment:
> I don't believe this is optional right? as without the text entity, we do not get any sort of custom away status?

Reply:
It's optional. If you don't use the text entity, you won't get a _custom_ status; you'll get a hardcoded status.